overlord/snapstate/handlers.go

```diff
@@ -897,6 +897,7 @@ func (m *SnapManager) doPreDownloadSnap(t *state.Task, tomb *tomb.Tomb) error {
 
 // asyncRefreshOnSnapClose asynchronously waits for the snap the close, notifies
 // the user and then triggers an auto-refresh.
+// It must be called with a lock on st.
```

Comment:
we usually formulate this as "// The caller should be holding the state lock."

Reply:
I'll rewrite it.

Reply:
Fixed.